components/camel-undertow/src/main/java/org/apache/camel/component/undertow/DefaultUndertowHost.java

```diff
@@ -37,6 +38,7 @@
     private CamelRootHandler rootHandler;
     private Undertow undertow;
     private String hostString;
+    private boolean http2Enabled;
```

Comment:
Would it not be better if `http2Enabled` was part of `UndertowHostOptions`?

It would be more consistent with how other Undertow server options are configured on the builder object.

Reply:
I agree